### PR TITLE
Add typings for create-keyframe-animation

### PR DESCRIPTION
**Pull request by fengyangfifa**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## create-keyframe-animation ([<kbd>unpkg</kbd>](https://unpkg.com/browse/create-keyframe-animation@latest/))
was missing the following properties:
1. hasAnimation
1. runAnimation
1. unregisterAnimation
1. getAnimationCSS
1. registerAnimation
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1da9c209a24ffadcade58deb3de72ab57c1e4e65
</p>

File: types/create-keyframe-animation/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    /**
+     * Register animation
+     * @param opts Animation execution configuration
+     */
+    registerAnimation(opts: RegisterAnimationConfig): void;
+}
+
+declare const animations: Animations;
+export default animations;
```
Comment (peterblazejewicz) [inline types/create-keyframe-animation/index.d.ts:94]: I this is an old school CJS module, with direct functions exports, so:
```ts
...
export function hasAnimation(name: string): boolean;
...

export function runAnimation<T extends OptsType>(els: ElsType, opts: T): Promise<T> | void;
```

File: types/create-keyframe-animation/create-keyframe-animation-tests.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import animations from "create-keyframe-animation";
```
Comment (peterblazejewicz) [inline types/create-keyframe-animation/create-keyframe-animation-tests.ts:1]:
```suggestion
import * as animations from "create-keyframe-animation";
```
ditto (about exports)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@fengyangfifa thanks!

**Comment by fengyangfifa** [comment]
> Ready to merge

**Comment by fengyangfifa** [comment]
@peterblazejewicz thanks!

**Comment by fengyangfifa** [comment]
Ready to merge

**Pull request merged**